hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/repository/EntityRepository.java

```diff
@@ -21,16 +21,23 @@
  */
 
 import java.util.Optional;
+import org.springframework.data.jpa.repository.Modifying;
 import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
+import org.springframework.transaction.annotation.Transactional;
 
 import com.hedera.mirror.common.domain.entity.Entity;
 
+@Transactional
 public interface EntityRepository extends CrudRepository<Entity, Long> {
 
     @Query(value = "select id from entity where alias = ?1 and deleted <> true", nativeQuery = true)
     Optional<Long> findByAlias(byte[] alias);
 
     @Query(value = "select id from entity where evm_address = ?1 and deleted <> true", nativeQuery = true)
     Optional<Long> findByEvmAddress(byte[] evmAddress);
+
+    @Modifying
+    @Query(value = "update entity set type = 'CONTRACT' where id in (:ids)", nativeQuery = true)
```

Comment:
nit, can be a bit faster if we fix entity type for a lot of mis-labeled contract entities since we have an index on (id, type):

```suggestion
    @Query(value = "update entity set type = 'CONTRACT' where id in (:ids) and type <> 'CONTRACT'", nativeQuery = true)
```

Reply:
Great idea, I have updated the statements.